src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const { convertToCase } = require('./convertToCase/index');
+const { validateParameters } = require('./validateRequest');
+const http = require('http');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = normalizedURL.pathname.substring(1);
+    const queryParams = new URLSearchParams(normalizedURL.search);
+    const targetCase = queryParams.get('toCase');
+
+    const paramsErrors = validateParameters(originalText, targetCase);
+
+    if (paramsErrors.length) {
+      res.statusCode = 400;
+      res.write(JSON.stringify({ errors: paramsErrors }));
+      res.end();
```

Comment:
Same about status message and here you can provide JSON to the `end()` function too